### ISSUE TITLE
Why using CrossEntropyLoss

### ISSUE THREAD

**Issue by glmanhtu** [yu4u/age-estimation-pytorch#19]
Hello @yu4u 

May I ask what is the reason of using CrossEntropyLoss instead of MSE/MAE ? From my understanding, age can go beyond 100, and therefore, MSE/MAE loss should be the better criteria for this problem ?
